src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -226,6 +227,10 @@ export class BaseEditCourseForm extends React.Component {
     this.setCollapsible(true);
   }
 
+  filterCourseTypes2U(courseType) {
```

Comment:
If we remove "2U" from the object and parameter names we can avoid confusion in the future, making this a truly generic feature.

```suggestion
  filterCourseTypes(courseType) {
```